cpp/arcticdb/async/async_store.hpp

```diff
@@ -22,227 +22,263 @@ namespace arcticdb::async {
 template<class ClockType = util::SysClock>
 class AsyncStore : public Store {
 public:
-    AsyncStore(
-        std::shared_ptr<storage::Library> library,
-        const arcticdb::proto::encoding::VariantCodec &codec) :
-        library_(std::move(library)),
-        codec_(new arcticdb::proto::encoding::VariantCodec{codec}) {
+    AsyncStore(std::shared_ptr<storage::Library> library, const arcticdb::proto::encoding::VariantCodec& codec)
+        : library_(std::move(library)),
+          codec_(new arcticdb::proto::encoding::VariantCodec{codec})
+    {
     }
 
-    folly::Future<entity::VariantKey> write(
-        stream::KeyType key_type,
+    folly::Future<entity::VariantKey> write(stream::KeyType key_type,
```

Comment:
I prefer: once wrapping starts, the first argument should be wrapped as well.

Reply:
Not possible with clang format as of today.